soc/ti/k3/am6x/Kconfig.soc

```diff
@@ -29,9 +35,14 @@ config SOC_AM6442_M4
 	bool
 	select SOC_SERIES_AM6X_M4
 
+config SOC_J721E_R5
+	bool
+	select SOC_SERIES_AM6X_R5
+
 config SOC_SERIES
 	default "am6x" if SOC_SERIES_AM6X
 
 config SOC
 	default "am6234" if SOC_AM6234_M4 || SOC_AM6234_A53
 	default "am6442" if SOC_AM6442_M4
+	default "j721e" if SOC_J721E_R5
```

Comment:
only just noticed the difference here, these need to match with the cpucluster from the soc.yml file so if that's `main_r5f0_0` then this need to be `config SOC_J721E_MAIN_R5F0_0` not sure if @tejlmand has a comment on this

Reply:
Sure, updated.